src/components/Card/Card.jsx

```diff
@@ -5,10 +5,18 @@ import CardImage from '../CardImage/CardImage';
 import CardFooter from '../CardFooter/CardFooter';
 
 const Card = ({ img, title, subtitle, label }) => {
+  const deleteHandler = () => {};
+  const editHandler = () => {};
+
   return (
     <div className={styles.Card}>
       <CardImage img={img} />
-      <CardFooter title={title} subtitle={subtitle} label={label} />
+      <CardFooter
```

Comment:
I just wonder whether there is a need to have `CardImage`, `CardFooter`, etc as separate components? From my perspective, those components can't be reused somewhere else to add something to the overall composition, they are needed only inside `Card` component and to have them as a separate ones can create only mess with number of components withing project.
What are your thoughts on this matter?

Reply:
In my opinion such separation increases readability

Reply:
> In my opinion such separation increases readability
Hi Oleksandr,
In the current project Card is unique component. There is no need to create separate files with CardImage and CardFooter components that will not be reused.

Reply:
I agree, but still I don't want to have all of this tags in the Card component, so I split it into several components